English/CommunitySources.md

```diff
@@ -1,35 +1,35 @@
 # Community sources
 
-## C++ общее
+## C++ General
 
 - [CppReference](https://en.cppreference.com)
 - [CPlusPlus](https://www.cplusplus.com/reference)
 - [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines)
-- [Рабочая группа C++ - представители комитета по стандартизации языка в России](https://stdcpp.ru/)
-- [Новости от комитета стандартизации С++ (ENG)](https://isocpp.org/)
+- [C++ working group - representatives of the language standardization committee in Russia](https://stdcpp.ru/)
```

Comment:
This link is not interesting for non-Russian readers. I propose to remove it or replace it with English alternatives.